tests/Integration/Behaviour/Features/Context/Domain/Product/ProductImageFeatureContext.php

```diff
@@ -329,7 +329,7 @@ public function assertProductHasNoImages(string $productReference): void
     }
 
     /**
-     * @Then product :productReference should have no images for shop :shops
+     * @Then product :productReference should have no images for shops :shops
```

Comment:
typo fix - the following step expects array in implementation.

Reply:
```suggestion
     * @Then product :productReference should have no images for shop(s) :shops
```
Actually you can make the s optional so it can be used with one or several references

Reply:
I didn't think its worth complicating the code for such minor wording "improvement" thing, we (including you 😄 ) used this kind of wording in many places already, why it started bothering you only now? 😄

Reply:
That's just a suggestion, I actually don't mind ^^ But the `(s)` is already used in several places and it's convenient because it allows using the two forms easily Not blocking though! But since you were modifying the sentence I just suggested this

Reply:
I think it works for both singular and multiple shops now too, so its just a syntax sugar ✌️